lab/app/views/samples/edit.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= form_with(model: @sample, local: true) do |form| %>
+       
+    <table>
+    <tr>
+        <td>
+        <%= form.label :sample_type %><br>
+        <%= form.text_field :sample_type %>
```

Comment:
é melhor que isso fosse uma lista (ex.: sangue, swab). O rails tem uma forma simples de fazer isso usando um [helper](https://guides.rubyonrails.org/form_helpers.html#select-boxes-for-dealing-with-models)

Se tiver dificuldades me avisa